client/import.go

```diff
@@ -208,6 +217,12 @@ func (c *Client) Import(ctx context.Context, reader io.Reader, opts ...ImportOpt
 	}
 
 	for i := range imgs {
+		if imgs[i].Labels == nil {
+			imgs[i].Labels = make(map[string]string)
+		}
+		for k, v := range iopts.imageLabels {
+			imgs[i].Labels[k] = v
+		}
 		img, err := is.Update(ctx, imgs[i], "target")
```

Comment:
Sorry, in this case, `imgs[i].Labels` is always nil. We just needs to assign it.

So, I think that code should like:

```go
fieldsPath := []string{"target"}
if iopts.imageLabels != nil {
      fieldsPath = append(fieldsPath, "labels")
      imgs[i].Labels =  iopts.imageLabels
}
img, err := is.Update(ctx, imgs[i], fieldsPath...)
```

You can test it with `images import  --local  -`

Reply:
Thank you, I've followed your comment and added some tests for this flag. 🙏